fast/stages/02-networking-vpn/data/firewall-rules/dev/rules.yaml

```diff
@@ -0,0 +1,33 @@
+# skip boilerplate check
+
+allow-dataflow-load-ingress-traffic:
+  description: "Allow traffic on Cloud Dataflow subnet"
+  direction: INGRESS
+  action: allow
+  sources: []
+  ranges:
+    - 10.10.0.0/24
+    - 10.10.1.0/24    
+  targets: []
+  use_service_accounts: false
+  rules:
+    - protocol: tcp
+      ports:
+        - 12345
+        - 12346
+
+allow-composer-health-checks:
```

Comment:
this is already done at the folder level via hierarchical firewal policies AFAIK